quickstart/go/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Use the offical Golang image to create a build artifact.
+# This is based on Debian and sets the GOPATH to /go.
+# https://hub.docker.com/_/golang
+FROM golang:1.12 as builder
+
+# Copy local code to the container image.
+WORKDIR /go/src/github.com/knative/docs/helloworld
```

Comment:
Update (usually `/app` or `/code` or something).

Reply:
sorry, I don't understand what this means